src/backend/aspen/cli/db.py

```diff
@@ -242,6 +249,8 @@ def create_phylo_run(
         )
         workflow.inputs = [aligned_gisaid_dump]
         workflow.template_args = json.loads(builds_template_args)
+        if tree_name:
+            workflow.name = tree_name
```

Comment:
how is this passed to what the user see on tree table?

Reply:
This name gets copied over to the PhyloTrees table when the tree is complete, and that's the default name for the tree in the UI:
https://github.com/chanzuckerberg/aspen/blob/trunk/src/backend/aspen/app/views/phylo_trees.py#L113